com.unity.multiplayer.mlapi/Runtime/com.unity.multiplayer.mlapi.runtime.asmdef

```diff
@@ -1,10 +1,12 @@
 {
     "name": "Unity.Multiplayer.MLAPI.Runtime",
     "rootNamespace": "MLAPI",
-    "references": [],
```

Comment:
is this change necessary? (unsafe is necessary but this one?)

Reply:
This change was made by the editor when I changed things via the UI.